lib/screens/more_details_page.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:project_eureka_flutter/components/eureka_appbar.dart';
+
+import 'package:project_eureka_flutter/screens/home_page.dart';
+
+class MoreDetails extends StatefulWidget {
+  @override
+  _MoreDetailsState createState() => _MoreDetailsState();
+}
+
+class _MoreDetailsState extends State<MoreDetails> {
+  String id; // user id
+
+  @override
+  void initState() {
+    initGetQuestionDetails(id);
+    super.initState();
+  }
+
+  void initGetQuestionDetails(String id) {}
+
+  Column _headerStack() {
+    return Column(
+      children: [
+        Container(
+          height: 0,
+        ),
+        _profileNameAndIcon(),
+        _answerQuestion(),
+      ],
+    );
+  }
+
+  Row _profileNameAndIcon() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        CircleAvatar(
+          radius: 55.0,
+          backgroundColor: Colors.grey[300],
+          child: CircleAvatar(
+            radius: 40.0,
+            backgroundImage: AssetImage('assets/images/dn.jpg'),
+          ),
+        ),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              "Question Title here",
+              style: TextStyle(fontSize: 25.0, fontWeight: FontWeight.bold),
+              textAlign: TextAlign.left,
+            ),
+            Text(
+              "Technology - Computer Hardware",
+              style: TextStyle(fontSize: 14.0, fontWeight: FontWeight.w300),
+              textAlign: TextAlign.left,
+            ),
+            Text(
+              "Asked : 2 days ago",
+              style: TextStyle(fontSize: 14.0, fontWeight: FontWeight.bold),
+              textAlign: TextAlign.left,
+            ),
+            Padding(
+              padding:
+                  const EdgeInsets.only(left: 0, top: 30, right: 0, bottom: 0),
+              child: RichText(
+                text: TextSpan(
+                    // set the default style for the children TextSpans
+                    style: TextStyle(
+                        fontSize: 14.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.black),
+                    children: [
+                      TextSpan(
+                        text: 'Attached media: ',
+                      ),
+                      TextSpan(
+                          text: 'Image1',
+                          style: TextStyle(
+                              color: Colors.blue,
+                              decoration: TextDecoration.underline)),
+                      TextSpan(
+                        text: '  ',
+                      ),
+                      TextSpan(
+                          text: 'Image2',
+                          style: TextStyle(
+                              color: Colors.blue,
+                              decoration: TextDecoration.underline)),
```

Comment:
Is there any chance to make a function that will return different number of image links based on some simulated list of images? Users may have different numbers of images, so I believe it cannot be fixed number, which is 2 in this case.

Reply:
I agree with Max here, you can possibly use a `for each` that will make these TextSpan the length of how many of these media there are. 
Some resources: https://stackoverflow.com/questions/50441168/iterating-through-a-list-to-render-multiple-widgets-in-flutter

Reply:
Done